### PR TITLE
Implement-multiline-strings

### PR DESCRIPTION
**Pull request by marysiuk**
Some edits.

### CODE REVIEW

**Comment by al-serebrov** [comment]
Some edits? Very descriptive (no).
Please add some explanations on what was changed :)

**Comment by marysiuk** [comment]
1. Made some indents using:
`print(' ')`
`\n`
2. Corrected grammatics.

_*If you have a more rational way to do indents I'll be glad to hear it_

File: mfp.py [hunk 1]
```diff
@@ -1,28 +1,30 @@
-print('Hi dude!')
+print('\nHi dude!')
+print('Whats your name?\n')
 
-print('Whats your name?')
 usr_name = input()
-print('So, ' + usr_name + ' is my homie.')
+print(' ')
```
Comment (al-serebrov) [inline mfp.py]: You can do just `print()` instead

File: mfp.py [hunk 2]
```diff
@@ -1,28 +1,30 @@
-print('Hi dude!')
+print('\nHi dude!')
+print('Whats your name?\n')
```
Comment (al-serebrov) [inline mfp.py]: Actually you can rework this piece using multi-line string as:
```python
hello_message = '''
Hi dude!
What's your name?
'''
print(hello_message)
```
This should give you the same result, but you don't have to care about line breaks `\n`

File: mfp.py [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
-print('So, ' + usr_name + ' is my homie.')
+print(' ')
 
-print('How old are you?')
+print('So, ' + usr_name + ' is my homie.')
+print('How old are you?\n')
 
 while True:
     usr_age = input()
+    print(' ')
```
Comment (al-serebrov) [inline mfp.py]: The same just `print()`

File: mfp.py [hunk 4]
```diff
@@ -1,28 +1,30 @@
...
     if int(usr_age) < 100:
 
         years_left = str(100 - int(usr_age))
 
         print(
             'Holy cow, {0}, you have {1} years left! \nIf I had {1} years I\'ll learn \
-programming till I die!'.format(
+programming everyday!\n'.format(
```
Comment (al-serebrov) [inline mfp.py]: Here you can use multi-line string too if you want ofc (it's ok as is)

**Review by al-serebrov — APPROVED** [review]
LGTM
Please check comments.

**Comment by marysiuk** [comment]
Please, check my modified mfp.py.

**Comment by al-serebrov** [comment]
Outstanding! Looks much cleaner! 👍 
Feel free to merge.

**Comment by al-serebrov** [comment]
Just one more quick remark :)
Branch name is `modify` and pull request name is `modify <script_name>` which is not very descriptive too. Try to name branches and give pull request titles in a short and explanatory manner like naming branch as "implement-multiline-strings" or something like this but shorter.

**Pull request merged**